### DISCUSSION TITLE
Flet black screen

### DISCUSSION THREAD

**Discussion by FranciscoLucas1** [flet-dev/flet#2125] [Q&A]
### Question

I am experiencing a display error in the framework. When I enter the name and click the button to go to the next page, where there should be a message "Hello username" as programmed in the code, the page refreshes and displays a black screen. I asked a colleague to test the code, and everything seems to work fine for them, but for me, only the black screen appears

I'm using Windows 10, Flet 0,12.0 and Python 3.12

![Screenshot 1](https://github.com/flet-dev/flet/assets/89021048/91c884f6-d0ab-4989-b271-778f08dad6a8)
![Screenshot 2](https://github.com/flet-dev/flet/assets/89021048/a4921d82-6226-4154-87e0-7b9611a9c3b7)

### Code sample

```python
import flet as ft

def main(page: ft.Page):

    def login(e):
        if not in_name.value:
            in_name.error_text = "Please insert your name"
            page.update()
        else:
            page.clean()
            page.add(ft.Text(f"Hello {in_name.value}"))
            page.update()


    
    in_name = ft.TextField(label = "Insert a name")
    button = ft.ElevatedButton("Entrar", on_click = login)


    page.add(in_name, button)

ft.app(target = main)
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
That's strange.
It works on my end as expected.
